### PR TITLE
Updating Translations 1906 2019-5-31

### PR DESCRIPTION
**Pull request by BrightspaceGitHubReader**

### CODE REVIEW

File: lang/zh.json [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+   "noNeedToFinish" : "If you don't need to finish {count, plural, one {it} other {them}}, no problem.",
+   "openInNewTab" : "在新的 Web 浏览器选项卡中打开链接。",
+   "openNew" : "在新窗口中打开",
+   "openOnedriveFile" : "在新的浏览器选项卡中打开 OneDrive 文件。",
+   "openUnsecureContent" : "您的浏览器认为本内容发布在一个不太安全的位置。别担心，只要在新窗口中打开内容即可。",
+   "showMissed" : "Show Me What I Missed",
+   "showModules" : "Show {numberOfModules} Modules",
+   "undisplayableContent" : "本内容无法在此查看器中显示",
+   "viewThisActivity" : "查看该活动。",
+   "youMissedThese" : "You missed these activities:"
```
Comment (iFun) [inline lang/zh.json]: I think the translation is missing

Comment (sarahboyd) [reply lang/zh.json]: yaaaa

Comment (sarahboyd) [reply lang/zh.json]: we'll have to just put in what we got for now, not sure when we'll get the rest

Comment (iFun) [reply lang/zh.json]: ok

**Pull request merged**